```
Whitespace check found 1 issues:
--
  | doc/src/manual/modules.md:338 -- trailing whitespace
  | make: *** [Makefile:106: check-whitespace] Error 1
```

--- a/doc/src/manual/modules.md
+++ b/doc/src/manual/modules.md
@@ -335,7 +335,7 @@ add (generic function with 1 method)
 
 julia> """
            add(x, y)
-       
+
        This sums up `x` and `y` and returns the result.
        """
        arithmetic.add

--- a/doc/src/manual/modules.md
+++ b/doc/src/manual/modules.md
@@ -333,12 +333,6 @@ Main.arithmetic
 julia> @eval arithmetic add(x, y) = $(+)(x, y)
 add (generic function with 1 method)
 
-julia> @doc "`add(x,y)` adds `x` and `y` together." arithmetic.add
-Main.arithmetic.add
-
-help?> arithmetic.add
-  add(x,y) adds x and y together.
-
 julia> arithmetic.add(12, 13)
 25
 ```

--- a/doc/src/manual/modules.md
+++ b/doc/src/manual/modules.md
@@ -333,18 +333,11 @@ Main.arithmetic
 julia> @eval arithmetic add(x, y) = $(+)(x, y)
 add (generic function with 1 method)
 
-julia> """
-           add(x, y)
-
-       This sums up `x` and `y` and returns the result.
-       """
-       arithmetic.add
+julia> @doc "`add(x,y)` adds `x` and `y` together." arithmetic.add
 Main.arithmetic.add
 
 help?> arithmetic.add
-  add(x, y)
-
-  This sums up x and y and returns the result.
+  add(x,y) adds x and y together.
 
 julia> arithmetic.add(12, 13)
 25

